OWNERS_ALIASES

```diff
@@ -62,3 +62,45 @@ aliases:
   - tsorya
   - osherdp
   - omertuc
+  coreos-approvers:
```

Comment:
kelvinfan001, arithx, and zonggen are no longer on the team.  @ashcrow, should lorbuschris, sosiouxme, and jupierce be included in these lists?